src/test/java/it/gov/pagopa/ranking/connector/rest/pdv/PdvErrorDecoderExt.java

```diff
@@ -0,0 +1,22 @@
+package it.gov.pagopa.ranking.connector.rest.pdv;
+
+import feign.Response;
+
+public class PdvErrorDecoderExt extends PdvErrorDecoder{
+
+    private static int invocationCount;
+
+    @Override
+    public Exception decode(String methodKey, Response response) {
+        invocationCount++;
+        return super.decode(methodKey, response);
+    }
+
+    public int getInvocationCount() {
```

Comment:
This could be static, this way it's not necessary to create this object in order to read it
```suggestion
    public static int getInvocationCount() {
```